walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -25,12 +25,36 @@
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     List<TimeRange> availableTimes = new ArrayList<>();
     
+    if (!request.getOptionalAttendees().isEmpty()) {
+      handleQuery(events, request, availableTimes, true);
```

Comment:
In cases where you're passing values which aren't immediately clear (`true`), please do the following:

```java
handleQuery(events, request, availableTimes, /* includeOptional= */  true);
```